dist/interfaces/JsonRpcMethods.d.ts

```diff
@@ -9,6 +9,7 @@ export declare enum JSON_RPC_REQUEST_METHOD {
     GET_STAKERS = "get_stakers",
     GET_FILTERED_SC_OUTPUT_EVENT = "get_filtered_sc_output_event",
     EXECUTE_READ_ONLY_REQUEST = "execute_read_only_request",
```

Comment:
This doesn't exist anymore and has been replaced by "execute_read_only_bytecode"